javascript/javascript3/week3/shopping_cart/main.js

```diff
@@ -0,0 +1,93 @@
+const userSection = document.querySelector('#user');
+const productsList = document.querySelector('#productsList');
+
+class Product {
+    constructor(name, price) {
+        this.name = name;
+        this.price = price;
+    }
+
+    convertToCurrency(currency) {
+       const currencyRate = {
+            'krone' : 1,
+            'dollar' : 0.16,
+            'euro' : 0.13,
+            'hryvnia' : 4.66
+        }
+        return this.price * currencyRate[currency];
+    }
+}
+
+const plant = new Product("plant", 50);
+console.log(plant.convertToCurrency('dollar'));
+
+class ShoppingCart {
+    constructor() {
+        this.products = [];
+    }
+
+    addProduct(product) {
+        this.products.push(product);
+    }
+
+    removeProduct(product) {
+        this.products = this.products.filter(el => el.name !== product.name);
+    }
+
+    searchProduct(productName) {
+        return this.products.filter(el => el.name === productName);
+    }
+
+    getTotal() {
+        let total = 0;
+        for (let i = 0; i < this.products.length; i++) {
+            total += this.products[i].price;
+        }
+        return total;
+    }
+
+    renderProducts() {
+        for (let i = 0; i < this.products.length; i++) {
+            const productName = document.createElement('h2');
+            productName.innerText = this.products[i].name;
+            productsList.appendChild(productName);
+            const productPrice = document.createElement('p');
+            productPrice.innerText = this.products[i].price;
+            productsList.appendChild(productPrice);
+        }
+    }
+
+    getUser() {
```

Comment:
don't you want to pass 'user' as a parameter? It's hard coded and less interesting.